DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: urltools
 Type: Package
 Title: Vectorised Tools for URL Handling and Parsing
-Version: 1.5.1
+Version: 1.5.1.9000
```

Comment:
Aye, but there will be a major change for 1.6.0, and releasing to CRAN before those changes are made is unlikely since I don't like bothering them too often, sooo ;)